src/go/k8s/config/e2e-test/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
```

Comment:
Wow I made a mistake. We don't need this.

Some background:

I can only envision that after @ivotron resolve https://github.com/vectorizedio/redpanda/issues/690 then this will be obsolete. Never the less current situation force us to overwrite `imagePullPolicy` to `Never`. Kuttl loads our image to the kind cluster, but we are using `latest` tag. 

> imagePullPolicy is omitted and either the image tag is :latest or it is omitted: Always is applied.


> NOTE: The Kubernetes default pull policy is IfNotPresent unless the image tag is :latest or omitted (and implicitly  :latest) in which case the default policy is Always. IfNotPresent causes the Kubelet to skip pulling an image if it already exists. If you want those images loaded into node to work as expected, please:
>
> don't use a :latest tag
and / or:
>
> specify imagePullPolicy: IfNotPresent or imagePullPolicy: Never on your container(s).
> See Kubernetes imagePullPolicy for more information.

In our case https://github.com/vectorizedio/redpanda/blob/dev/src/go/k8s/config/default/manager_auth_proxy_patch.yaml#L23

REF
- https://github.com/vectorizedio/redpanda/blob/dev/src/go/k8s/Makefile#L3
- https://kubernetes.io/docs/concepts/containers/images/#updating-images
- https://kind.sigs.k8s.io/docs/user/quick-start/

Reply:
Still there are some problems. I will comeback with the answers shortly.

Reply:
Thank you for pointing the problem I fixed this. You will see in the next push.